Natasha Yakovleva/widget/styles.css

```diff
@@ -0,0 +1,104 @@
+body{
+	background-image: url('bg.png');
+	background-size: cover;
+}
+
+.main{
+	padding-left: 50px;
+	padding-top: 60px;
+}
+
+.date{
+	margin-top: -365px;
+	margin-left: 414px;
+	display: flex;
+
+}
+
+.crt{
+	margin-top: -35px;
+	margin-left: 530px;
+	display: flex;
+	flex-direction: column;
+}
+
+.degrees{
+	display: inline-flex;
+}
+
+.degree{
+	padding-left: 5px;
+}
+
+.location{
+	padding-top: 15px;
+	padding-left: -1px;
+}
+
+.wind{
+	padding-top: 20px;
+	display: inline-flex;
+}
+
+.wind_picture{
+	padding: 3px;
+}
+
+.weather{
+	padding-left: 40px;
+	padding-right: 50px;
+	margin-top: 80px;
+	display: inline-flex;
```

Comment:
Keep in mind order of properties, group them by type
1 - layout (position, display)
2 - anything related to box model (width/height/margin/padding)
3 - styling for element (color, background, box-shadow)
4 - anything related to font
5 - other stuff, it also includes overflow, z-index